### PR TITLE
Add a python-fastapi usecase

### PR DESCRIPTION
**Pull request by ToneVDB**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          echo "$(pwd)/bin" >> $GITHUB_PATH
+          curl -o bin/sbot -L https://github.com/restechnica/semverbot/releases/download/v$SEMVERBOT_VERSION/sbot-linux-amd64
+          chmod +x bin/sbot
+
+      - name: make venv
+        run: python -m venv venv
+
+      - name: activate venv
+        run: . venv/bin/activate
+
```
Comment (shiouen) [inline .github/workflows/main.yaml:30]: would be good to have a `sbot update version` step as well I think

File: python/fastapi/.python-version [hunk 2]
```diff
@@ -0,0 +1 @@
+3.9.5
```
Comment (shiouen) [inline python/fastapi/.python-version:1]: nvm this comment

**Review by shiouen — COMMENTED** [review]
left some comments

File: .github/workflows/main.yaml (hunk 1 above)
Comment (ToneVDB) [reply .github/workflows/main.yaml:30]: Yeah, missing a poetry install here as well

Comment (ToneVDB) [reply .github/workflows/main.yaml:30]: updated

**Comment by ToneVDB** [comment]
fun story - never name your project in the `pyproject.toml` file the same as a dependency, it wont install/lock/... do anything, but wont throw an error either 🙃

**Review by shiouen — APPROVED** [review]
lgtm

**Pull request merged**